radixdlt-java/src/main/java/com/radixdlt/client/application/translate/tokens/UnknownTokenException.java

```diff
@@ -0,0 +1,30 @@
+package com.radixdlt.client.application.translate.tokens;
+
+public class UnknownTokenException extends RuntimeException {
```

Comment:
Do we want to document stuff like exceptions now or at some later time?

Reply:
I'll add some documentation.